glusterfsd/src/glusterfsd-mgmt.c

```diff
@@ -8,67 +8,27 @@
    cases as published by the Free Software Foundation.
 */
 #include <stdio.h>
-#include <sys/types.h>
-#include <sys/wait.h>
 #include <stdlib.h>
-#include <signal.h>
 
 #include <glusterfs/glusterfs.h>
-#include <glusterfs/dict.h>
-#include <glusterfs/gf-event.h>
-#include <glusterfs/defaults.h>
+#include <glusterfs/statedump.h>
+#include <glusterfs/syscall.h>
+#include <glusterfs/monitoring.h>
 
 #include "rpc-clnt.h"
-#include "protocol-common.h"
 #include "glusterfsd-messages.h"
 #include "glusterfs3.h"
 #include "portmap-xdr.h"
-#include "xdr-generic.h"
-
 #include "glusterfsd.h"
 #include "rpcsvc.h"
 #include "cli1-xdr.h"
-#include <glusterfs/statedump.h>
-#include <glusterfs/syncop.h>
-#include <glusterfs/xlator.h>
-#include <glusterfs/syscall.h>
-#include <glusterfs/monitoring.h>
 #include "server.h"
 
 static gf_boolean_t is_mgmt_rpc_reconnect = _gf_false;
-int need_emancipate = 0;
-
-int
-glusterfs_mgmt_pmap_signin(glusterfs_ctx_t *ctx);
-int
-glusterfs_volfile_fetch(glusterfs_ctx_t *ctx);
-int
-glusterfs_process_volfp(glusterfs_ctx_t *ctx, FILE *fp);
-int
-emancipate(glusterfs_ctx_t *ctx, int ret);
-int
-glusterfs_process_svc_attach_volfp(glusterfs_ctx_t *ctx, FILE *fp,
-                                   char *volfile_id, char *checksum,
-                                   dict_t *dict);
-int
-glusterfs_mux_volfile_reconfigure(FILE *newvolfile_fp, glusterfs_ctx_t *ctx,
-                                  gf_volfile_t *volfile_obj, char *checksum,
-                                  dict_t *dict);
-int
-glusterfs_process_svc_attach_volfp(glusterfs_ctx_t *ctx, FILE *fp,
-                                   char *volfile_id, char *checksum,
-                                   dict_t *dict);
-int
-glusterfs_process_svc_detach(glusterfs_ctx_t *ctx, gf_volfile_t *volfile_obj);
 
-gf_boolean_t
-mgmt_is_multiplexed_daemon(char *name);
+static gf_boolean_t need_emancipate = _gf_false;
 
 static int
-glusterfs_volume_top_perf(const char *brick_path, dict_t *dict,
-                          gf_boolean_t write_test);
-
-int
 mgmt_cbk_spec(struct rpc_clnt *rpc, void *mydata, void *data)
```

Comment:
nit: I *think* it can be static

Reply:
@dmantipov missed this one, do consider in any of the future PRs. Thanks.